server/sonar-server/src/main/java/org/sonar/server/component/es/ProjectMeasuresIndex.java

```diff
@@ -91,6 +99,19 @@ private static QueryBuilder toValueQuery(MetricCriterion criterion) {
       default:
         throw new IllegalStateException("Metric criteria non supported: " + criterion.getOperator().name());
     }
+  }
 
+  private QueryBuilder createAuthorizationFilter() {
+    Integer userLogin = userSession.getUserId();
+    Set<String> userGroupNames = userSession.getUserGroups();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "getUserGroups"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)